website_react/src/components/inputs/InputBoolean.jsx

```diff
@@ -0,0 +1,32 @@
+const InputBoolean = (props) => {
+    return (
+        <div className="form-input-div" key={props.index}>
+            <label className="input-label">{props.formFields.title}</label>
+            <div>
+                <input
+                    type="radio"
+                    id={"true" + props.index}
```

Comment:
are we/the form is using the id?

Reply:
no, but it's just a good practice to add `id` to form inputs